packages/patternfly-4/react-core/src/components/Tabs/Tabs.js

```diff
@@ -29,13 +29,11 @@ const propTypes = {
   /** enables Secondary Tab styling */
   isSecondary: PropTypes.bool,
   /** aria-label for the left Scroll Button */
-  leftScrollAriaLabel: PropTypes.string,
+  'aria-label-left-scroll': PropTypes.string,
   /** aria-label for the right Scroll Button */
-  rightScrollAriaLabel: PropTypes.string,
+  'aria-label-right-scroll': PropTypes.string,
```

Comment:
I like this approach for naming. I'd like to suggest we do this for some of the other components that currently only have one aria-label prop. For example, in the ContextSelector component, we have a search button that gets labeled with the `aria-label` prop. In this case, I'd suggest we use `aria-label-search`. This would let us later update the component to include other things that would use the attribute, and avoid potential confusion by having one prop as `aria-label` and another as `aria-label-[descriptive text]`.

I do think there are cases where having just `aria-label` is fine for the most granular components, like the Button component, where that's the one and only text string we would define. Or where the string defined in `aria-label` is applied to the parent element of a more complex component (like a fancy modal). But other components like Chip or ContextSelector, where it's applied to something buried inside the component, I'd suggest we make the prop name more specific.

Reply:
I made all of the aria props that aren't just passed to the parent <div> longer names.

Reply:
Related to the discussion in slack, this should change to `ariaLabelRightScroll` (and same for the left scroll)